WalletWasabi/Tor/Http/Helpers/HttpRequestUriBuilder.cs

```diff
@@ -58,11 +58,5 @@ public HttpRequestUriBuilder(UriScheme uriScheme, string host, string path = "")
 
 			Path = path;
 		}
-
-		public Uri BuildUri(string path)
```

Comment:
My MSVS reports that the `HttpRequestUriBuilder` is actually unused too. I deleted the file and it builds ok.

Reply:
Deleted.